tex/measure/martingale.tex

```diff
@@ -192,11 +211,57 @@ \section{Conditional expectation}
 
 The way that this is typically handled in measure theory
 is to use the notion of sub-$\sigma$-algebra that we defined.
-Let $\SF$ be a sub-$\sigma$-algebra which captures the information
-This means we create a function assigning the
-``conditional expectation'' to \emph{every} point $\omega \in \Omega$,
-which is measurable with respect to $\SF$.
-\todo{give the example}
+
+But first, we should explain what $\EE(X \mid X+Y)$ means first --- why are we conditioning on
+another random variable instead of an event?
```

Comment:
Is this "common high school knowledge"? (I sort of didn't know this before)

Reply:
I don't think so.